main_loop.h

```diff
@@ -453,7 +453,7 @@ class MainLoop {
       output_encoder_bias_ = calibration_.output_encoder_bias;
       error_mask_.all = calibration_.error_mask.all == 0 ? ERROR_MASK_ALL : (calibration_.error_mask.all & ERROR_MASK_ALL);
       output_encoder_dir_ = param_.output_encoder.dir == 0 ? 1 : param_.output_encoder.dir;
-      torque_sensor_dir_ = param_.torque_sensor.dir == 0 ? 1 : param_.torque_sensor.dir;
+      torque_sensor_dir_ = calibration_.torque_sensor.dir == 0 ? 1 : calibration_.torque_sensor.dir;
```

Comment:
This would be ok but I would potentially still like to add back  a param torque sensor dir so that we wouldn't have to set this value in every calibration

Reply:
It could be the product of both